pvlib/tests/iotools/test_psm3.py

```diff
@@ -21,7 +23,20 @@
     'Temperature Units', 'Pressure Units', 'Wind Direction Units',
     'Wind Speed', 'Surface Albedo Units', 'Version']
 PVLIB_EMAIL = 'pvlib-admin@googlegroups.com'
-DEMO_KEY = 'DEMO_KEY'
+
+
+@pytest.fixture(scope="module")
+def nrel_api_key():
+    """"""
```

Comment:
NIT: remove empty docstring, or add meaningful message

Reply:
Missed this, thanks for highlighting. 

I've included some verbage about what is going on for our future selfs.